stats/statsview.cpp

```diff
@@ -719,13 +719,13 @@ void StatsView::QuartileMarker::updatePosition()
 		      x + quartileMarkerSize / 2.0, y);
 }
 
-StatsView::RegressionLine::RegressionLine(double a, double b, QPen pen, QtCharts::QChart *chart, StatsAxis *xAxis, StatsAxis *yAxis) :
-	item(new QGraphicsLineItem(chart)),
+StatsView::RegressionLine::RegressionLine(double a, double b, double width, QBrush brush, QtCharts::QChart *chart, StatsAxis *xAxis, StatsAxis *yAxis) :
+	item(new QGraphicsPolygonItem(chart)),
 	xAxis(xAxis), yAxis(yAxis),
-	a(a), b(b)
+	a(a), b(b), width(width)
 {
-	item->setZValue(ZValues::chartFeatures);
```

Comment:
Sorry - I think now you removed one line too much: The zValue still has to be set.

If that is fixed, consider it approved by me. I can resolve the conflict, if you want.

Reply:
You are right. I had experimented with the z value to make the dots stand out more but that was not effective. So I wanted to remove that again and accidentally removed the whole line.